Packs/SymantecEndpointSecurity/Integrations/SymantecEndpointSecurity/SymantecEndpointSecurity_test.py

```diff
@@ -0,0 +1,339 @@
+            },
+            [
+                {
+                    "uuid": "123",
+                    "log_time": "2024-10-09T12:34:56Z",
+                    "time": "2024-10-09T12:34:56Z",
+                },
+                {
+                    "uuid": "456",
+                    "log_time": "2024-10-09T12:34:56.789Z",
+                    "time": "2024-10-09T12:34:56.789Z",
+                },
+                {
+                    "uuid": "789",
+                    "log_time": "2024-10-09T12:34:55.789Z",
+                    "time": "2024-10-09T12:34:55.789Z",
+                },
+            ],
+            [],
+            id="Event time is equal to or less than last_event_time",
+        ),
+        pytest.param(
+            {
+                "events_suspected_duplicates": ["123"],
+                "latest_event_time": "2024-10-09T12:34:56Z",
+            },
+            [
+                {
+                    "uuid": "123",
+                    "log_time": "2024-10-09T12:34:56Z",
+                    "time": "2024-10-09T12:34:56Z",
+                },
+                {
+                    "uuid": "456",
+                    "log_time": "2024-10-09T12:34:56.789Z",
+                    "time": "2024-10-09T12:34:56.789Z",
+                },
+            ],
+            [
+                {
+                    "uuid": "456",
+                    "log_time": "2024-10-09T12:34:56.789Z",
+                    "time": "2024-10-09T12:34:56.789Z",
+                    "_time": "2024-10-09T12:34:56.789Z",
+                }
+            ],
+            id="Events time is equal to last_event_time but one of them not include in suspected duplicates",
+        ),
+        pytest.param(
+            {
+                "events_suspected_duplicates": ["123"],
+                "latest_event_time": "2024-10-09T12:34:56Z",
+            },
+            [
+                {
+                    "uuid": "456",
+                    "log_time": "2024-10-09T12:35:56.789Z",
+                    "time": "2024-10-09T12:35:56.789Z",
+                },
+            ],
+            [
+                {
+                    "uuid": "456",
+                    "log_time": "2024-10-09T12:35:56.789Z",
+                    "time": "2024-10-09T12:35:56.789Z",
+                    "_time": "2024-10-09T12:35:56.789Z",
+                }
+            ],
+            id="Events time is greater than last_event_time",
+        ),
+    ],
+)
+def test_filter_duplicate_events(
+    integration_context: dict[str, str],
+    events: list[dict[str, str]],
+    expected_filtered_events: list[dict[str, str]],
+):
+    """
+    Given:
+        - A list of events with timestamps
+    When:
+        - The `filter_duplicate_events` function is called
+    Then:
+        - Ensure that a list of the events that are not duplicates is returned
+    """
+    filtered_events = filter_duplicate_events(events, integration_context)
```

Comment:
instead of passing the whole expected events, we can (won't insist here - but please note for future PRs) pass the id, making it easier to debug.